RELEASE_GUIDE.md

```diff
@@ -44,48 +67,142 @@ considering what effects this may have:
+  #   The new version should be based on the PVP,
+  #   according to what changed since the previous release, #   not on the actual compiler version.
+  #   See https://pvp.haskell.org/
+$EDITOR lib/purescript-cst/purescript-cst.cabal
 
-  - The `prerelease` field in `app/Version.hs`, if updating the prerelease
-    field
+  # 3b. Bump the version
+  # 3c. If it changed, update the `purescript-cst`
+  #   version to match the new one
+$EDITOR purescript.cabal
 
-  - The `version` field in `npm-package/package.json`
+  # 3d. If making a pre-release,
+  # change the `prerelease` string.
+  # Otherwise, it should be "" for all
+  # other releases.
+$EDITOR app/Version.hs
 
-  - The version to install in the `postinstall` script in `package.json`
+  # 3e. Update the table
+$EDITOR lib/purescript-cst/README.md
 
-  - If `purescript-cst` has changed at all since the last release:
+  # 3f. Change the "version" field
+  # 3g. In the `postinstall` script,
+  #   change the version.
+$EDITOR npm-package/package.json
 
-      - The `version` field in `lib/purescript-cst/purescript-cst.cabal` (note
-        that the new version should be based on the PVP, according to what
-        changed since the previous release, and not on the actual compiler
-        version)
+git add \
+  lib/purescript-cst/purescript-cst.cabal \
+  lib/purescript-cst/README.md \
+  purescript.cabal \
+  app/Version.hs \
+  npm-package/package.json
+git commit -m "Bumped versions for upcoming release"
 
-      - The versions table in `lib/purescript-cst/README.md`,
+# 4. Create the PR either manually or
+#   via the `gh` CLI tool
+gh pr create
+# a. Make the `base` repo `purescript`
+# b. Push your branch to your fork
+# c. Continue on the web
 
-      - The version bounds for `purescript-cst` in `purescript.cabal`
+# 5. Push any other changes to the PR
+#   until it gets merged.
+```
 
-- Create a release from the releases tab in GitHub and copy in the release
-  notes. This will also create a tag, which will kick off a CI build, which
-  will upload prebuilt compiler binaries to the release on GitHub when it
-  completes. (If the CI build fails, binaries can also be built locally and
-  manually uploaded to the release on GitHub)
+### Make the GitHub Release
 
-- Publish to Hackage:
+Once the above PR has been merged, create a release from the releases tab in GitHub, copying and pasting the version's corresponding section in the CHANGELOG.md file into the release as its release notes. This will also create a tag, which will kick off a CI build. The CI build will upload prebuilt compiler binaries to the release on GitHub when it completes.
 
-  - change to the `lib/purescript-cst` directory and run `stack upload .`
+If the GitHub-release-triggered CI build fails, the binaries will need to be built locally and manually uploaded to the release on GitHub.
 
-  - Finally, run `stack upload .` from the repo root directory.
+INSTRUCTIONS TO BE WRITTEN.
```

Comment:
Would this be 
```bash
# if Linux
CI_RELEASE="true" ./bundle/build.sh "Linux"
# if Mac
CI_RELEASE="true" ./bundle/build.sh "macOS"
# if Windows
CI_RELEASE="true" ./bundle/build.sh "Windows"
```

Reply:
It would be along those lines, but you'll also need to make sure you've built a release version of the compiler first as well (so that `purs --version` doesn't say it's a development build). In the name of saving effort, since this scenario might not even happen, I'm tempted to say let's not bother writing instructions. Since we are the maintainers of the CI workflow/release script, we should be able to figure out how to run it if need be.